lizard_auth_server/views.py

```diff
@@ -94,11 +94,31 @@ def portals(self):
         else:
             return self.profile.portals.all()
 
+    @cached_property
+    def oidc_userconsent(self):
+        """
+        Returns UserConsent of the user that is logged in.
+        UserConsent couples foreign key from user and OIDC clients
+        to give consent for logging in on the OIDC client.
```

Comment:
Deze zin loopt niet helemaal.

Reply:
Done. Klopt het inhoudelijk?

Reply:
"Returns" => "Return" (gebiedende wijs i.p.v. beschrijven).

En: ik denk dat het UserConsents, dus meervoud, is?